src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -3131,6 +3131,28 @@ internal WebView Assign(WebView webView)
             return ret;
         }
 
+        [Obsolete("Do not use this, that is deprecated in API13.")]
```

Comment:
```suggestion
        // [Obsolete("Do not use this, that is deprecated in API13.")]
```

Remove this Obsolete. It will show compile warning messages to application developer.